### ISSUE TITLE
few-shot 代码跑不通

### ISSUE THREAD

**Issue by zhaoym55** [MCG-NJU/AWT#2]
您好：
请教一下如何跑通few-shot 代码。我这边训练完成后测试报错，调试发现测试集好像没有做数据增强
def parse_batch_test(self, batch):
        input = batch["img"]
        label = batch["label"]
        input = torch.stack(input, dim=0) # aug_time x batch_size x 3 x 224 x 224
        input = torch.transpose(input,0,1).contiguous() # batch_size x aug_time x 3 x 224 x 224
        input = input.to(self.device)
        label = label.to(self.device)
        return input, label
这个input从testloader出来时batchsize*3*224*224.请问这个问题怎么解决呢？

**Comment by zyuhan1999** [comment]
Hi,

I noticed your issue was closed. If you still have any questions, please feel free to reach out.

Best,
Yuhan

**Comment by xiaojieren11** [comment]
请问您解决了吗，我也遇到了相同的问题。
